.cursor/skills/agentic-workflow-foundation/scripts/test_apply_kit_init.py

```
@@ -0,0 +1,276 @@
+        errors.append(f"--check: expected exit 0, got {rc}\n{log}")
+    if 'name: "[要確認]"' not in out:
+        errors.append("--check: manifest was modified (should be dry-run)")
+
+    # 10. Idempotency: apply twice → same result
+    rc1, out1, _ = _run(MANIFEST_FIXTURE, INIT_VALID)
+    with tempfile.TemporaryDirectory(prefix="test-idempotent-") as tmp:
+        m = Path(tmp) / "manifest.yaml"
+        i = Path(tmp) / "init.yaml"
+        m.write_text(out1, encoding="utf-8")
+        i.write_text(INIT_VALID, encoding="utf-8")
+        r = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)